src/lib/updateGrantsRecipients.js

```diff
@@ -339,6 +360,14 @@ export async function processFiles(hashSumHex) {
 
       await Promise.all(grantUpdatePromises);
 
+      // Automate CDI linking to preceding recipients
+      const cdiGrantsToLink = await Grant.unscoped().findAll({
+        where: { regionId: 13, endDate: { [Op.gte]: '2021-03-17' } },
```

Comment:
Im trying to think if there is a way to limit the number of records coming back here. But there might not be...

Reply:
Why is there a need to limit it?

Reply:
Simply to limit the amount of unnecessary processing but it might not be possible/needed.

Reply:
Good thinking, but if a grant is marked with region 13, that means it needs an update, if possible. We could restrict to only the grants that don't have null grantId's, but there are only 3 grants right now.